src/cpu/aarch64/jit_uni_reorder.cpp

```diff
@@ -1678,6 +1696,10 @@ struct jit_uni_reorder_kernel_f32_t : public kernel_t, public jit_generator {
         UNROLL_INST(fcvtzs, VReg4S, tmp, tmp);
     }
 
+    void cvt_v_f32_bf16(const size_t startIdx, const size_t regNum) {
+        UNROLL_INST2(bfcvt, ZRegH(i), p_lsb_128 / T_m, ZRegS(i));
```

Comment:
```suggestion
        UNROLL_INST2(bfcvtn, VReg4H(i), VReg4S(i));
```
`bfcvtn` of the smaller instruction set is better.